api/apiclient_test.go

```diff
@@ -1241,24 +1241,6 @@ func (s *apiclientSuite) TestLoginCapturesCLIArgs(c *gc.C) {
 	c.Assert(request.CLIArgs, gc.Equals, `this is "the test" command`)
 }
 
-func (s *apiclientSuite) TestLoginIncompatibleClient(c *gc.C) {
```

Comment:
Instead of removing this, should we actually invert it and ensure that clients can connect if they have a matching facade version but not a matching major version?